src/Microsoft.DotNet.ProjectModel/LibraryDescription.cs

```diff
@@ -40,6 +42,7 @@ public class LibraryDescription
         public string Path { get; }
         public IEnumerable<LibraryRange> Dependencies { get; }
         public bool Compatible { get; }
+        public string PackagesDirectoryPath { get; }
```

Comment:
For some reason, I find the plural "Packages" hard to reason about here.  It is really the "SubPath" under the packages directory.  And it is also hard to reason between "Path" and "PackagesDirectoryPath".

Also, since this property only really makes sense on "PackageDescription" objects, I wonder if it would be better if it were only at that class level.